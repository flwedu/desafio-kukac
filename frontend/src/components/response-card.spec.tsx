import { render } from "@testing-library/react";
import ResponseCard from "./response-card";
describe("Response Card component tests", () => {
  test("Should not render the component", () => {
    const { container } = render(<ResponseCard results="" />);

    expect.assertions(1);
    expect(container).toMatchInlineSnapshot("<div />");
  });

  test("Should render the component properly", () => {
    const { container } = render(<ResponseCard results="Test Result" />);

    expect.assertions(1);
    expect(container).toMatchSnapshot();
  });
});

export {};
