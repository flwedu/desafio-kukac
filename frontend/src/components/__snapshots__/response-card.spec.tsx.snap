// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Response Card component tests Should render the component properly 1`] = `
<div>
  <div
    class="response card shadow rounded"
  >
    <p>
      Esse Ã© o resultado: 
    </p>
    <textarea
      id="results"
      name="results"
      readonly=""
    >
      Test Result
    </textarea>
  </div>
</div>
`;
